assets/src/css/core-components.css

```diff
@@ -43,12 +43,12 @@
 .amp .components-button:not(.components-panel__body-toggle):focus:not(:disabled),
 .amp .components-button:not(.components-panel__body-toggle):hover {
 	box-shadow: none;
-	color: var(--brand);
+	color: var(--amp-settings-color-brand);
 	text-decoration: none;
 }
 
-.amp .components-button:focus:not(:disabled) {
-	outline: 1px solid var(--brand);
+.amp.amp .components-button:focus:not(:disabled) {
```

Comment:
Typo?

```suggestion
.amp .components-button:focus:not(:disabled) {
```

Reply:
Not a typo, just needed a tad more specificity on this selector.